templates/index.html

```diff
@@ -25,12 +25,13 @@ <h2>Items</h2>
         <!-- For loop to List the todo items here -->
         {% for item in items %}
          <!-- Form to add a toggle status button -->
-          <li>
-            <form action="/{{item.id}}" method="post"> 
-            <label for="title">{{item.title}}</label> -
-            <!-- Button styled to look like a link--> 
-            <button type="submit" name=sub class="btn btn-link">{{item.status}}</button>
-          </form>
+         <li>
+            <form action="/{{item.id}}/completed" method="post"> 
+              <label for="title">{{item.title}}</label>
+              {% if item.status != 'Completed' %}
+              <button type="submit" name=sub class="btn btn-outline-primary">Mark Completed</button>
+              {% endif %}
+            </form>
```

Comment:
It would be nicer UI to display the status of each item or split them into different lists.